docs/samples/v1beta1/sklearn/v1/sklearn-mixedtype-model/sklearn-mixedtype.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-mixedtype"
+spec:
+  predictor:
+    sklearn:
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/mixedtype"
+      image: "suresh1233/sklearnserver:latest"
```

Comment:
```suggestion
      image: "kserve/sklearnserver:latest"
```

Reply:
or maybe can simply remove the image field as the image is going to be published when 0.8 is released

Reply:
sure @yuzisun

Reply:
updated!